### ISSUE TITLE
[Bug]: V8 ChoiceGroup Fluent2WebDarkTheme color contrast issue

### ISSUE THREAD

**Issue by gfisher-vmsft** [microsoft/fluentui#28508]
### Library

React / v8 (@fluentui/react)

### System Info

```shell
Fluent V8 ChoiceGroup with Fluent2WebDarkTheme
```

### Are you reporting Accessibility issue?

yes

### Reproduction

https://codepen.io/v-grfisher/pen/bGQaQGM

## Actual Behavior
Color contrast issue with React V8 choice group icons using Fluent2WebDarkTheme
![image](https://github.com/microsoft/fluentui/assets/125615874/9d031265-20b7-47db-b81b-e4f08b7ce1ed)

Title: WCAG 1.4.3: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (#ChoiceGroupLabel4-week)
Tags: Accessibility, WCAG 1.4.3, color-contrast

Issue: Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds (color-contrast - https://accessibilityinsights.io/info-examples/web/color-contrast)

Target application: Fabric Example Pen - https://codepen.io/v-grfisher/pen/bGQaQGM

Element path: #result;#ChoiceGroupLabel4-week

Snippet: <span id="ChoiceGroupLabel4-week" class="ms-ChoiceFieldLabel">Week</span>

Related paths: 
  #result;.ms-ChoiceField.root-114.ms-ChoiceField--icon:nth-child(2)

How to fix: 
Fix any of the following:
  Element has insufficient color contrast of 4.39 (foreground color: #8a8886, background color: #242424, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1

Environment: Microsoft Edge version 114.0.1823.79

====

This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

## Expected Behavior
No color contrast issues

### Requested priority

Blocking

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.
